inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1,120 +1,236 @@
 ---
 title: "Mortality survey"
-output: github_document
+output: word_document
 ---
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE,
+                      warning = FALSE,
+                      error = TRUE)
 library(dplyr)
 library(knitr)
-library(epireports)
+library(sitrep)
 library(survey)
+library(ggplot2)
 ```
 
-```{r, message=FALSE}
-# study_data_raw <- readxl::read_excel("...")
-study_data_raw <- outbreaks::fluH7N9_china_2013
-
-study_data_cleaned <- study_data_raw %>%
+```{r read_data, message = FALSE}
+# use read_data stuff from outbreak?
+
+
+## Read data ------------------------------------
+# CSV file
+# linelist_raw <- rio::import("linelist.csv")
+#
+# Excel file
+# to read in a specific sheet use "which"
+# linelist_raw <- rio::import("linelist.xlsx", which = "Sheet1")
+#
+# Stata data file
+# linelist_raw <- rio::import("linelist.dat")
+#
+# For password protected Excel file 
+# use the excel.link package 
+# library(excel.link)
+# linelist_raw <- excel.link::xl.read.file("linelist.xlsx",
+#                                          xl.sheet = "Sheet1",
+#                                          password = askpass::askpass(prompt = "please enter file
+#                                                                      password"))
+
+# linelist_raw <- outbreaks::fluH7N9_china_2013
+linelist_raw <- gen_data(dictionary = "Mortality", varnames = "column_name",
+                         numcases = 500)
+
+study_data_cleaned <- linelist_raw %>%
   # some cleaning
```

Comment:
in the cholera template - cleaning is broken down by variable types. May be worth doing here too?
Although all in one mutate is quite nice actually!

Reply:
this can be done in the end, i think. for now i just expanded it whenever a new variable was needed.